File: InforceShortener.Data/Models/UrlRecord.cs
namespace InforceShortener.Data.Models
{
    public class UrlRecord : IModel
    {
        public int Id { get; set; }

        public string OriginalUrl { get; set; }

        public string ShortUrl { get; set; }

        public DateTime CreatedDate { get; set; }


        public virtual User User { get; set; }
    }
}

File: Angular/inforce-shortener-app/src/app/shared/models/urlRecord.ts
export class UrlRecord {
    constructor(id: string, originalUrl: string, shortUrl: string, createdDate: string, userName: string, userRole: string)
    {
        this.id = id;
        this.originalUrl = originalUrl;
        this.shortUrl = shortUrl;
        this.createdDate = createdDate;
        this.userName = userName;
        this.userRole = userRole;
    }

    id!: string;
    originalUrl!: string;
    shortUrl!: string;
    createdDate!: string;
    userName!: string;
    userRole!: string;
}